generator/src/lib.rs

```diff
@@ -2620,7 +2632,11 @@ pub fn generate_handle(handle: &vkxml::Handle) -> Option<TokenStream> {
     };
     Some(tokens)
 }
-fn generate_funcptr(fnptr: &vkxml::FunctionPointer, has_lifetimes: &HashSet<Ident>) -> TokenStream {
+fn generate_funcptr(
+    fnptr: &vkxml::FunctionPointer,
+
```

Comment:
is the empty line on purpose ?

Reply:
It is not something that is relevant when discussing a proof-of-concept that already had only a 20% chance of being successful.